### ISSUE TITLE
Change where replies to DMP review email are directed

### ISSUE THREAD

**Issue by sjDCC** [DMPRoadmap/roadmap#2006]
When researchers have submitted a plan for review and receive the notification that this has been completed, they often hit reply to thank the uni or ask questions. These replies come to the DMPonline (or DMPTool) helpdesk rather than being directed to the uni as we issued the email. Please could we change this so replies to plan review emails go to the uni helpdesk or contact specified on the org details page.

**Comment by sjDCC** [comment]
Hi @johnpinto1 I can't validate this. I have switched the contact email for two unis to my address to test it, but on both Roadmap-stage and DMPonline-test when I submit a plan for feedback, it no longer gets added to the notifications panel so I can't 'complete' the review and issue the email I need to reply to.

Can you check the logic of the DMP review submission please as I think something has gone wrong earlier in the process. We can raise a separate ticket for this or it may be related

**Comment by sjDCC** [comment]
Quick follow up to this:

As an admin I received an email to the plan to review and could follow the link to get access. This allows me to comment but since the table isn't displaying on the admin side, I can't access the 'complete review' button and consequently verify where emails go if I (as a user) reply to this notification.

The act of submitting a plan for review did not add it to a notifications table on the admin 'plans' page as it should. Should we raise this as a new ticket or keep it here so the two issues can be UAT-ed together once the full workflow is in place again?

Permissions to view and comment are being correctly assigned.

**Comment by johnpinto1** [comment]
@sjDCC @xsrust and @briri I investigate the issues raised in last comment this as part of this ticket with input from the developers on what may have caused it.

**Comment by xsrust** [comment]
@sjDCC did you submit the review from the same account as you are an org-admin?
If so, it auto-hides it from the UI.

After testing with a new user account to submit the plan for review, this is working and sending from the RDM email account
